src/Middleware/OutputCaching/test/OutputCacheMiddlewareTests.cs

```diff
@@ -334,44 +334,44 @@ public void ContentIsNotModified_IfNoneMatch_MatchesAtLeastOneValue_True()
     [Fact]
     public void StartResponseAsync_IfAllowResponseCaptureIsTrue_SetsResponseTime()
     {
-        var clock = new TestClock
+        var time = new TestTime
```

Comment:
Waht about having a simple reusable fake `TimeProvider` that would have `SetUtcNow(DateTimeOffset)` and `Advance(TimeSpan)`. Would we reuse it in other places in the solution?

Reply:
There might be a plan for runtime to provide something for this
https://github.com/dotnet/runtime/issues/36617#issuecomment-1492119130

Reply:
I'm surprised @sebastienros since he's the one implementing the FakeTimeProvider literally as we speak.

Reply:
I assumed this work here could just use a simpler one, just by adding a couple of properties to the current one, until we have a reusable one that provides everything. Also I am too not sure about the plan of having one in a public package from dotnet/runtime.